packages/next-adapter/src/customize/index.ts

```diff
@@ -44,7 +44,7 @@ export async function runAsync({
   for (const customization of manifest) {
     const enabled = await customization.onEnabledAsync({ projectRoot, force });
     values.push({
-      name: customization.name,
+      title: customization.name,
```

Comment:
Has the new version of Typescript allowed you to find this bug? (was this even a bug? I'm just curious). If not, this seems unrelated.

Reply:
Yes, this resulted in a compilation error with TS 3.8. (Not sure why it wasn't an error before.)